src/pages/onboarding/components/commonOnboarding/Step계열선택.tsx

```diff
@@ -40,22 +39,18 @@ const ItemWrapper = styled.li`
   align-items: center;
 
   height: 4.8rem;
-  padding: 1rem 1.5rem;
-  border-radius: 8px;
-
-  background-color: ${({ theme }) => theme.colors.grayScaleWG};
+  padding: 1rem 0;
 `;
-const Item = styled.div`
+
+const Text = styled.p`
   display: flex;
   gap: 1rem;
   align-items: center;
 
+  color: ${({ theme }) => theme.colors.grayScaleDG};
   ${({ theme }) => theme.fonts.Title1_SB_16};
 `;
 
-const Icon = styled.i`
-  width: 2.3rem;
-  height: 2.3rem;
-
-  background-color: ${({ theme }) => theme.colors.grayScaleBlack};
+const CheckItemIcon = styled(CheckItemIc)<{ $isActive: boolean }>`
+  fill: ${({ theme, $isActive }) => ($isActive ? theme.colors.Blue : theme.colors.grayScaleDG)};
```

Comment:
```suggestion
const CheckItemIcon = styled(CheckItemIc)<{ $isActive: boolean }>`
  fill: ${({ theme, $isActive }) => ($isActive ? theme.colors.Blue : theme.colors.grayScaleLG2)};
```

p1) 여기 default 체크 색상 BG 아니고 LG2 인거같아요! 

![image](https://github.com/user-attachments/assets/035814c9-ce28-4799-a88b-cd39685cf067)

![image](https://github.com/user-attachments/assets/b84729ab-6580-4344-a0ba-7218923154e9)

Reply:
헉 그러네요 !! 디테일 감사합니당 ㅠㅠㅠㅜ!!!